src/services/point.service.ts

```diff
@@ -15,8 +16,39 @@ export class PointService {
       return;
     }
 
+    // cache the #1 point holder before points are awarded
+    const [kingBeforeAward] = await this.getTopPoints(1);
+
     const userDoc = await this.getUserPointDoc(id);
     await PointsModel.updateOne(userDoc, { $inc: { numPoints: amount } });
+
+    const [kingAfterAward] = await this.getTopPoints(1);
+
+    // the king has changed!
+    if (kingBeforeAward.userID !== kingAfterAward.userID) {
+      // remove the role from the previous king
+      await this._guild.members.cache
+        .get(kingBeforeAward.userID)
+        ?.roles.remove(Constants.Roles.TacoKing);
```

Comment:
I dont think that works

If it doesnt you'll have to use `GuildService.getRole(Constants.Roles.TacoKing)` to get the actual Discord `Role` then pass that in